src/openrct2/config/Config.cpp

```diff
@@ -1001,7 +1001,8 @@ bool RCT1DataPresentAtLocation(const utf8* path)
 
 bool CsgIsUsable(const rct_gx& csg)
 {
-    return csg.header.total_size == RCT1::RCT1_LL_CSG1_DAT_FILE_SIZE && csg.header.num_entries == RCT1::RCT1_NUM_LL_CSG_ENTRIES;
+    return csg.header.total_size == RCT1::Limits::LL_CSG1_DAT_FileSize
+        && csg.header.num_entries == RCT1::Limits::Num_LL_CSG_Entries;
```

Comment:
Is this a limit, per se?

Reply:
Yeah i was unsure what to do with these constants. I'm somewhat contemplating another header just for special constants.